### DISCUSSION TITLE
Covid 19風雲再起

### DISCUSSION THREAD

**Discussion by wangtyphoon** [1101-GS4719-C/Python-Community-GS4719-C#939] [General]
covid19從發現至今已滿三年，在中國全面放開防疫的情況下，日本重新恢復對中國旅客的檢疫，且我觀察美國CDC的疫情報告XBB變種已非常快的速度取代了BA.5和後繼的BQ.1，但願今年冬天是最後一個被疫情所壟罩的季節

https://covid.cdc.gov/covid-data-tracker/#variant-proportions
![image](https://user-images.githubusercontent.com/109240416/209698128-65d3b73c-6974-4de1-8a89-c06dcf131820.png)
![image](https://user-images.githubusercontent.com/109240416/209698161-15d56185-442c-499c-a54f-8b09b8e31c2d.png)
![image](https://user-images.githubusercontent.com/109240416/209698185-beb85910-5330-4839-88c5-70727a5f49c6.png)

**Comment by peggy0329** [comment]
病毒真的變種的好快QQ
今年寒假想去打第四劑，希望疫情不要在打亂人們生活了

**Comment by Zoe06060606** [comment]
希望及使併毒變種對我們將來都不要有太大的影響 能像現在一樣好好生活

**Comment by Evan0979** [comment]
拜託疫情快結束...我都快忘記疫情前的生活是怎麼樣了嗚嗚嗚

**Comment by chia-kai-chang** [comment]
這些Visualization好漂亮，一張圖勝過千言萬語

**Comment by StevenWanglolz** [comment]
好好奇這些圖表是怎麼製作的喔~我有修一堂通識課期末要製作資訊圖表視覺化，也想做的這麼好看!

**Comment by Floriaand** [comment]
接近期末考了要戴好口罩好好保護身體

**Comment by CPCpyt** [comment]
雖然大家都有打了疫苗，但假如遇到新的變種可能也沒有抵抗力了。
最近在中國很多朋友突然就陽了，雖然中國解封，但我認為在經濟、人口上的隱患還是存在。

**Comment by CPCpyt** [comment]
有些台灣朋友這學期還是不幸二次染疫，
大家還是要注意自己的身體健康，多保重。
希望可以早日回復到過往的生活。
